www/src/pages/en.astro

```diff
@@ -0,0 +1,7 @@
+---
+const redirectUrl = "/";
```

Comment:
We can do this in the vercel.json:

https://vercel.com/docs/project-configuration#project-configuration/redirects

Reply:
I wanted to separate the deployment from the actual site but I guess it won't matter here, we're not changing Vercel for anything haha
The con of doing it in Vercel's file is that you can't really have this behavior in the dev env (does it matter at all?)

Reply:
It's fine I think?